src/helplist.tooltip.js

```diff
@@ -0,0 +1,20 @@
+export default {
```

Comment:
This is the right idea, but I'd probably put this file in `src/data/help.js` (I don't think there's any need to specify that they have to do with tooltips specifically)